packages/cozy-client/src/associations/index.js

```diff
@@ -27,14 +43,9 @@ export const create = (
     throw new Error(`Association ${name} already exists`)
   }
 
-  switch (type) {
-    case 'has-many':
-      return doctype === 'io.cozy.files'
-        ? new HasManyFilesAssociation(target, name, doctype, accessors)
-        : new HasManyAssociation(target, name, doctype, accessors)
-    case 'has-many-UNSAFE':
-      return new HasManyUNSAFEAssociation(target, name, doctype, accessors)
-    default:
-      throw new Error(`Can't handle ${type} associations`)
+  const cls = getClass(doctype, type)
+  if (!cls) {
+    throw new Error(`Can't handle ${type} associations`)
```

Comment:
Is it really possible to enter this condition, since we already `throw` in `getClass` if we don't return anything ?